Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+podTemplate(
```

Comment:
This is copied from https://github.com/ischemaview/cordova-plugin-background-download/blob/development/Jenkinsfile

I don't think any changes are needed here, but I could be wrong.

Reply:
looks good. thank you.